pcsx2/DebugTools/DebugInterface.h

```diff
@@ -81,8 +82,9 @@ class DebugInterface : public MemoryReader
 	virtual bool isValidAddress(u32 address) = 0;
 	virtual u32 getCycles() = 0;
 	virtual BreakPointCpu getCpuType() = 0;
-	[[nodiscard]] virtual SymbolGuardian& GetSymbolGuardian() const = 0;
-	[[nodiscard]] virtual std::vector<std::unique_ptr<BiosThread>> GetThreadList() const = 0;
+	virtual SymbolGuardian& GetSymbolGuardian() const = 0;
+	virtual SymbolImporter* GetSymbolImporter() const = 0;
```

Comment:
<rant> I'm a professional raw pointer hater (not really, I love pointer arithmetic). But this makes me with std::optional<T&> was legal C++ :( </rant>